test/OpenTelemetry.Tests/Metrics/AggregatorTestsBase.cs

```diff
@@ -238,6 +238,48 @@ public void MultiThreadedHistogramUpdateAndSnapShotTest()
         Assert.Equal(200, argsToThread.SumOfDelta + lastDelta);
     }
 
+    [Theory]
+    [InlineData("Microsoft.AspNetCore.Hosting", "http.server.request.duration")]
+    [InlineData("Microsoft.AspNetCore.Http.Connections", "signalr.server.connection.duration")]
+    [InlineData("Microsoft.AspNetCore.RateLimiting", "aspnetcore.rate_limiting.request_lease.duration")]
+    [InlineData("Microsoft.AspNetCore.RateLimiting", "aspnetcore.rate_limiting.request.time_in_queue")]
+    [InlineData("Microsoft.AspNetCore.Server.Kestrel", "kestrel.connection.duration")]
+    [InlineData("Microsoft.AspNetCore.Server.Kestrel", "kestrel.tls_handshake.duration")]
+    [InlineData("OpenTelemetry.Instrumentation.AspNetCore", "http.server.duration")]
+    [InlineData("OpenTelemetry.Instrumentation.Http", "http.client.duration")]
+    [InlineData("System.Net.Http", "http.client.connection.duration")]
+    [InlineData("System.Net.Http", "http.client.request.duration")]
+    [InlineData("System.Net.Http", "http.client.request.time_in_queue")]
+    [InlineData("System.Net.NameResolution", "dns.lookups.duration")]
+    public void HistogramBucketsDefaultUpdatesForSecondsTest(string meterName, string instrumentName)
```

Comment:
Add another unit test to check that a regular Histogram with `Unit` as seconds still uses the default Histogram bounds.

Reply:
Is this resolved? I don't see another unit test added.